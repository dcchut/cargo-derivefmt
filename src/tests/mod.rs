use crate::modify_source;
use textwrap::dedent;

#[test]
fn test_derive_ordering() {
    let mut source = dedent(
        r#"
        #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
        struct Wrapped<T>(T);
    "#,
    );

    modify_source(&mut source).unwrap();
    insta::assert_snapshot!(source);
}
