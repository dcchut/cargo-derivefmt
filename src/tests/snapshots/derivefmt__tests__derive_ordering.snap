---
source: src/tests/mod.rs
expression: source
---

#[derive( Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd )]
struct Wrapped<T>(T);

